specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2021-04-01/replicationusages.json

```diff
@@ -0,0 +1,208 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-04-01",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/replicationUsages": {
```

Comment:
>replicationUsages [](http://example.com/codeflow?start=128&length=17)

How are these created? Is this a proxy resource?

Reply:
I have just copied the existing swagger to the new api version. I am not aware of what was there in this, as I dont have any change in this , I just copy pasted it to the new version

Reply:
yup, thanks for the clarification.